packaging/suse/patches/0_change_versions.gem.patch

```diff
@@ -117,16 +117,16 @@ index 481374c0b290..cf5d577309dc 100644
        activesupport (>= 4.1.0)
      gravatar_image_tag (1.2.0)
      guard (2.13.0)
-@@ -158,7 +158,7 @@ GEM
-     jquery-rails (3.1.3)
-       railties (>= 3.0, < 5.0)
-       thor (>= 0.14, < 2.0)
+@@ -153,7 +153,7 @@ GEM
+     hike (1.2.3)
+     hirb (0.7.3)
+     i18n (0.7.0)
 -    json (1.8.3)
 +    json (1.8.1)
      json-schema (2.5.1)
```

Comment:
this green and red refer to the 0_change_versions.gem.patch , sorry I didn't realize it. Looks good.